keps/sig-apps/2232-suspend-jobs/README.md

```diff
@@ -385,78 +390,86 @@ field.
 
 * **Are there any tests for feature enablement/disablement?** No.
 
-<!-- Uncomment when targeting beta graduation
-
 ### Rollout, Upgrade and Rollback Planning
 
 _This section must be completed when targeting beta graduation to a release._
 
-* **How can a rollout fail? Can it impact already running workloads?**
-  Try to be as paranoid as possible - e.g., what if some components will restart
-   mid-rollout?
-
-* **What specific metrics should inform a rollback?**
-
-* **Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?**
+* **How can a rollout fail? Can it impact already running workloads?** Impact
+  to existing Jobs that previously didn't use this feature in alpha is
+  impossible. In workloads using the feature in an older version, suspended
+  Jobs may inadvertently be resumed (or Jobs may be inadvertently suspended) if
+  there are storage-related issues arising from components crashing
+  mid-rollout.
+
+* **What specific metrics should inform a rollback?** `job_sync_duration_seconds`
+  and `job_sync_total` should be observed. Unexpected spikes in the metric with
+  labels `result=error` and `action=pods_deleted` is potentially an indicator
+  that:
+    1. Job suspension is producing errors in the Job controller,
+    1. Jobs are getting suspended when they shouldn't be, or
+    1. Job sync latency is high when Job are suspended.
+  While the above list isn't exhaustive, they're signals in favour of rollbacks.
+
+* **Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path
+  tested?** <!-- I'll answer this after implementation.
```

Comment:
You can do the same thing I did here:
https://github.com/kubernetes/enhancements/pull/2538/files